docs/get-started.md

```diff
@@ -107,160 +107,143 @@ Welcome to Nebula Graph (Version 0.1)
 nebula>
 ```
 
```

Comment:
add one line: If you have any trouble in starting nebula service in docker, feel free to raise an issue to us.